config/config-builder/src/validator_config.rs

```diff
@@ -194,19 +194,16 @@ impl ValidatorConfig {
         );
         let discovery_set = vm_genesis::make_placeholder_discovery_set(&validator_set);
 
-        let genesis = Some(Transaction::UserTransaction(
-            vm_genesis::encode_genesis_transaction_with_validator(
-                &faucet_key,
-                faucet_key.public_key(),
-                &validator_swarm.nodes,
-                validator_set,
-                discovery_set,
-                self.template
-                    .test
-                    .as_ref()
-                    .and_then(|config| config.publishing_option.clone()),
-            )
-            .into_inner(),
+        let genesis = Some(vm_genesis::encode_genesis_transaction_with_validator(
```

Comment:
I think at some point, we're going to need to fashion a genesis builder much like we have for config builder, but I have a plan for that lol...